stacks-signer/src/main.rs

```diff
@@ -81,6 +81,12 @@ fn write_chunk_to_stdout(chunk_opt: Option<Vec<u8>>) {
 fn spawn_running_signer(path: &PathBuf) -> SpawnedSigner {
     let config = GlobalConfig::try_from(path).unwrap();
     let endpoint = config.endpoint;
+    warn!(
+        "Reminder: The signer is primarily designed for use with a local or subnet network stacks node. \
```

Comment:
@jferrant @hstove would you suggest any changes to the message? Also if we'll have a link for faq in the future, I could add that. Thanks